src/actions/index.tsx

```diff
@@ -25,37 +25,29 @@ import { ApolloClient } from '@apollo/client';
 
 import {
     /** Filters */
-    ADD_FILTER,
-    DELETE_FILTER,
-    SET_QUERY_STRING,
-    SET_OPTIONS_FOR_FILTERS,
     GATE_ARTIFACTS_BUMP_SEARCH_EPOCH,
     GATE_ARTIFACTS_SET_SEARCH_OPTIONS,
     /** Alerts */
-    POP_ALERT,
-    PUSH_ALERT,
     ActionPopAlert,
     ActionGASetSearchOptions,
     ActionGABumpSearchEpoch,
-    WAIVER_CREATE,
-    WAIVER_RESET_REPLY,
-    WAIVER_RESULT,
-    FETCH_USER,
+    IStateWaiver,
+    IStateAuth,
+    IStateFilters,
 } from './types';
-import { store } from '../reduxStore';
+import { AppDispatch } from '../reduxStore';
+import * as alertsSlice from '../reducers/alertsSlice';
```

Comment:
I wonder what is the meaning of the word `slice`.
Would it be better to replace it with `reducers` ?